pkgs/desktops/deepin/go-package/go-gir-generator/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, go
+, pkg-config
+, libgudev
+, gobject-introspection
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
FYI @Artturin

Reply:
reference： https://github.com/NixOS/nixpkgs/pull/46373

Reply:
only the gobject-introspection library is needed here https://github.com/linuxdeepin/go-gir-generator#installation so gir in buildInputs is correct